addon/props/model_props.py

```diff
@@ -86,6 +86,21 @@ class SOURCEOPS_ModelProps(bpy.types.PropertyGroup):
         default=False,
     )
 
+    transform_source: bpy.props.EnumProperty(
+        name='Transform Source',
+        description='Method of specifying $origin and $scale.\nEither manually specified in this panel, or via an object',
+        items=[
+            ('MANUAL', 'Manual Input', 'Specify the Transform manually in this panel'),
+            ('OBJECT', 'Object', 'Use an Object\'s transforms\nIf it is None, then falls back to Manual Input')
```

Comment:
It does not fall back to manual input, but instead uses no transforms.

Reply:
Oh and I always use trailing commas for multi-line lists, so I'd like you to do that too, for consistency.

Reply:
Oh whoops! Forgot to update that string.
I think I understand what trailing comma you're requesting, I presume it's at the end of the 2nd item in the list; for some reason I thought it was in the strings and got really confused.
...Must be the weather at the moment. 🤣

Reply:
Added trailing comma at end of L94 here.